charts/traction/values.yaml

```diff
@@ -17,7 +17,7 @@ acapy:
     repository: ghcr.io/bcgov/traction-plugins-acapy
     pullPolicy: IfNotPresent
     pullSecrets: []
-    tag: ''
+    tag: '0.3.1'
```

Comment:
This should stay empty.

Reply:
Ah, i see - I thought it would grab the value from here, but it picks up the chart value *unless* this is specified. Will revert.